chat-widget/src/components/notificationpanestateful/defaultProps/defaultChatDisconnectStyleProps.ts

```diff
@@ -0,0 +1,93 @@
+import { IChatDisconnectStyleProps } from "@microsoft/omnichannel-chat-components/lib/types/components/notificationpane/interfaces/chatdisconnect/IChatDisconnectStyleProps";
+
+export const defaultChatDisconnectStyleProps: IChatDisconnectStyleProps = {
+    generalStyleProps: {
```

Comment:
Please assign these default styles at the component level. Otherwise component users will have weird default NotificationPane

Reply:
Discussed offline -- default styles will be assigned at the widget level given current design.